HoneyBadgers.Logic/Services/ReportService.cs

```diff
@@ -16,7 +18,7 @@ public class ReportService: IReportService
         private readonly IAuthService _authService;
         private readonly ILogger<ReportService> _logger;
         private readonly IHttpClientFactory _httpClientFactory;
-        private readonly string _baseUrl = "https://localhost:5001/api";
+        private readonly string _baseUrl = "https://localhost:44313/api";
```

Comment:
nie zmieniaj tego, bo każdy to będzie musiał zmieniać, żeby odpalić report na porcie 5001 to musisz odpalić rest api za pomocą ctrl + f5

Reply:
Nie znałem skrótu, świadomie zmieniłem, ponieważ nie mogłem uruchomić na portach 5001. Już przywracam

Reply:
Możesz też przełączyć visuala, by odpalał HoneyBadgers.RestApi/WebApp zamiast IIS. Pozmianie możesz w solucji ustawić też by się odpalały 2 projekty na raz :)